controllers/src/main/java/io/syndesis/controllers/integration/online/ActivateHandler.java

```diff
@@ -46,7 +50,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ActivateHandler implements StatusChangeHandlerProvider.StatusChangeHandler {
+public class ActivateHandler implements StatusChangeHandler {
```

Comment:
Actually, this is the "DeployHandler" and should be renamed accordingly (refering to the proposal). What is missing is the "ResumeHandler" (opposite of "SuspendHandler").